### ISSUE TITLE
Screen readers fail to read some menu items in some context menus of OBS Studio

### ISSUE THREAD

**Issue by k-kolev1985** [obsproject/obs-studio#11402]
### Operating System Info

Windows 11

### OBS Studio Version

30.2.3

### OBS Studio Log URL

https://obsproject.com/logs/tx42qrOtCy224xfA

### Expected Behavior

The screen readers should properly report all menu items (e.g. its label, its state (inactive/unavailable), if it has a submenu).

### Current Behavior

The screen readers do not properly report some menu items (e.g. its label, its state (inactive/unavailable), if it has a submenu).

### Steps to Reproduce

1. Enable high-contrast mode in Windows 11, by doing the following:
1.1. Open Windows Settings (Windows+I).
1.2. Go to Accessibility ➡ Contrast themes.
1.3. From the "Contrast themes" combo-box, select any dark theme.
1.4. Activate the "Apply" button.
2. Launch a screen reader like NVDA or Narrator.
3. Launch OBS Studio.
4. Open OBS settings ➡ Appearance.
5. From the "Theme" dropdown, select "System".
6. Click the OK button.
7. Create a scene and add at least one source to it.
8. Invoke the context menu for the added source and use the up/down arrow keys to navigate between its menu items.

### Anything else we should know?

As the title states, screen readers like NVDA and Narrator fail to read some menu items in some context menus of OBS Studio. The one noticed is the context menu for a source in the list of sources in the main window of OBS Studio, but I don't know if it is the only one that is problematic. And I just found out, that the problem occurs only when using the high-contrast black theme in Windows, as I do use it.

**NOTE:** There are a few useful comments about this issue in a recently closed old ticket: https://github.com/obsproject/obs-studio/issues/3363.
